cgi-bin/DW/Controller/Talk.pm

```diff
@@ -875,20 +882,27 @@ sub preview_parent_args {
             subject     => $parentitem->subject_html,
             poster      => $poster,
             poster_name => $poster_name,
+            in_journal  => $in_journal,
+            admin_post  => $parentitem->prop('admin_post'),
             icon        => $userpic_tag->($parentitem),
-            time        => $parentitem->{datepost},       # convert?
+            time        => $parentitem->{datepost},
             url         => $parentitem->url,
             entry_url   => $entry->url,
         };
     }
     else {
         # Replying to entry
+
+        my $in_journal =
+            $entry->poster->user eq $entry->journal->user ? '' : $entry->journal->ljuser_display;
```

Comment:
Same as above, use the `equals` method.

Reply:
Ah, good call! I had a lingering suspicion that there was a method for that, but was having a hard time finding it for some reason.